WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m

```diff
@@ -52,6 +45,25 @@ + (instancetype)buttonWithFrame:(CGRect)frame buttonStyle:(WPBlogSelectorButtonS
     return button;
 }
 
+- (void)didMoveToSuperview
+{
+    [super didMoveToSuperview];
+    [self invertLayout];
+}
+
+/// Inverts the layout to show the image at the right side
+///
+- (void)invertLayout
```

Comment:
Nitpick, but could we rename this something like `invertLayoutIfNeeded` or `ifRequired`?

Reply:
Oh! But it always invert it 😆
The button will always be the inverted of the current user layout.

Maybe the comment is misleading?

Reply:
No, I get it now – sorry, I misunderstood what was happening. Perhaps just update the comment to `Inverts the layout to show the image on the trailing side`, as it won't always be `right` if we're in RTL. Thanks! :shipit:!